source/accessible-media/production-audio-description.html.erb.md

```diff
@@ -77,56 +82,44 @@ American Council of the Blind’s [Audio Description
 Project](http://www.acb.org/adp/).
 
 
-## Production workflow: audio descriptions (human-recorded narration)
-
-### Basic workflow for creating pre-produced audio descriptions:
-
-{::nomarkdown}
-<%= sample_start %>
-{:/nomarkdown}
+## Producing human-recorded narration
 
 1. **Watch the video**<br>
     Note opportunities for descriptive narration (i.e., pauses in
     narration or dialog into which descriptions can be inserted).
+
 2. **Write the description script**<br>
     Use a description-authoring tool, a word processor or any text
     editor to create the script that will be recorded by the narrator.
+
 3. **Record the descriptions**<br>
     Using audio-editing software, record the narrator reading the
     description script. [Learn more about recording audio
     descriptions](http://www.descriptionkey.org/technical_issues.html).
-    -   Alternatively, create a computer-generated rendering of the
-        description script using TTS software. [See examples of
-        pre-recorded TTS descriptions](http://ncamftp.wgbh.org/ibm/dvs/)
-        to learn whether or not they are appropriate for all types of
-        media presentations.
+
+    When recording the descriptions, it will pay to create the
```

Comment:
Added the list that came after the workflow with relevant information to be displayed inside the workflow, contextually.